packages/noco-docs/content/en/setup-and-usages/keyboard-maneuver.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Keyboard Maneuver'
+description: 'Keyboard shortcuts'
+position: 700
+category: 'Product'
+menuTitle: 'Keyboard Maneuver'
+---
+
+
+## Shortcuts
+|       Key | Behaviour               |
+|----------:|:------------------------|
+| `ALT` `t` | Insert new table        |
+| `ALT` `r` | Insert new row          |
+| `ALT` `c` | Insert new column       |
+| `ALT` `f` | Toggle fullscreen mode  |
+| `ALT` `i` | Invite a member to team |
+| `ALT` `,` | Open Settings menu      |
+
+
+## General maneuver on GRID
+|             Key | Behaviour                                                                            |
+|----------------:|:-------------------------------------------------------------------------------------|
+|         `⌘` `↑` | Jump to first record in this column (in same page)                                   |
+|         `⌘` `↓` | Jump to last record in this column (in same page)                                    |
+|         `⌘` `C` | Copy cell contents                                                                   |
+|         `Enter` | Switch cell in focus to EDIT mode; opens modal/picker if cell is associated with one |
+|           `Esc` | Exit cell EDIT mode                                                                  |
+|        `Delete` | Clear cell                                                                           |
+|         `Space` | Expand current row                                                                   |
+| `←` `→` `↑` `↓` | General cell navigation                                                              |
+|           `Tab` | Move to next cell horizontally; if on last cell, move to next row beginning          |
+
+
+## Cell-specific maneuver when in EDIT mode (data type dependent)
+|        Datatype        |         Key | Behaviour                          |
+|:----------------------:|------------:|:-----------------------------------|
+| Text / Numerical cells |     `←` `→` | Move cursor to the left / right    |
+|                        |     `↑` `↓` | Move cursor to the beginning / end |
+|     Single Select      |     `↑` `↓` | Move between options               |
+|                        |     `Enter` | Select option                      |
+|      Multi Select      |     `↑` `↓` | Move between options               |
+|                        |     `Enter` | Select / deselect option           |
+|        DateTime        |  `Ctrl` `;` | Select current date time           |
+|          Link          |     `↑` `↓` | Move between options               |
+|                        |     `Enter` | Link current selection             |
+|        Checkbox        |     `Enter` | Toggle                             |
+|         Rating         | `<0 ~ Max>` | Enter number to toggle rating      |
+
+
+## Expanded row maneuver
```

Comment:
prefer `Expanded Form` over `Expanded Row` as it is using `Expanded Form` in the doc.